test/extended/registry/signature.go

```diff
@@ -0,0 +1,120 @@
+package registry
+
+import (
+	"fmt"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	imagesutil "github.com/openshift/origin/test/extended/images"
+	exutil "github.com/openshift/origin/test/extended/util"
+
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+)
+
+var _ = g.Describe("[imageapis] image signature workflow", func() {
```

Comment:
Since it involves registry, there's no harm adding `[registry]` as well. Sorry I missed that earlier.